web/src/services/i18n/translations/en.yml

```diff
@@ -376,7 +376,7 @@ Failed to update account name.: Failed to update account name.
 Failed to update password.: Failed to update password.
 Successfully updated password!: Successfully updated password!
 Failed to change language.: Failed to change language.
-Failed to change theme.: Failed to change theme.
+Failed to change theme.classic.: ''
```

Comment:
might be wrong

Reply:
See other comment

Reply:
(the key change is following other comment.)

In this diff, the content is empty. Is it no problem?